src/css/popup.css

```diff
@@ -271,21 +271,63 @@ main-panel {
   font-weight: 700;
 }
 
+.panel-status {
+  display: flex;
+  justify-content: space-between;
+  padding: 8px 15px;
```

Comment:
I know we aren't technically using Protocol tokens, but it may be good practice to adhere to the values of the tokens. 

```suggestion
  padding: 8px 16px;
```

Reply:
Good call!